icu4c/source/i18n/unicode/msgfmt.h

```diff
@@ -245,8 +245,9 @@ class NumberFormat;
  *
  * <h4>Argument formatting</h4>
  *
- * <p>Arguments are formatted according to their type, using the <code>toString()</code>
- * methods. But there are a few types that are known to be locale-sensitive.
+ * <p>Arguments are formatted according to their type, using the using default
+ * <code>Formattable</code> for those types, unless otherwise specified.
```

Comment:
s/<code>Formattable</code>/formatters (no "code" style)

the Formattable gives the type and value, but the formatting is done by *formatters*

Reply:
Done